graql/language/undefine.feature

```diff
@@ -1337,8 +1327,8 @@ Feature: Graql Undefine Query
       """
     Then answer size is: 0
 
-  #TODO: Reenable when rules actually do something
-  @ignore
+
+  @ignore # TODO enable when we can do reasoning in a schema write transaction
```

Comment:
Wasn't this TODO-comment-on-same-line-as-@ignore syntax the issue that we recently had a PR submitted to fix because it's not valid Gherkin syntax?

Reply:
lol good point